### DISCUSSION TITLE
[terraform] Bad indent on new block

### DISCUSSION THREAD

**Discussion by draeron** [zed-industries/zed#17506] [Feature Requests]
Not sure why but this is _very_ annoying. The terraform plugin makes a bad indent when creating a new block.

![image](https://github.com/user-attachments/assets/8faa24a7-6b77-4ed0-9013-26b1896ca941)

Not sure if this is related but saw this issue in tree-sitter

https://github.com/tree-sitter-grammars/tree-sitter-hcl/issues/15

Anyone with the same issue?

**Comment by SomeoneToIgnore** [comment]
There's https://github.com/zed-industries/zed/pull/17200 that seems related, can you test it and see if it changes anything for you?

**Reply by milosgajdos** [reply]
This is still broken for me on `0.155.2`.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
